### ISSUE TITLE
'new' expression, whose target lacks a construct signature, implicitly has an 'any' type

### ISSUE THREAD

**Issue by jet-lee-m** [steelbrain/php-serialize#230]
Hi,

i cannot deploy my code because i get an error in unserialize.ts.

```
node_modules/php-serialize/src/unserialize.ts:16:17 - error TS7009: 'new' expression, whose target lacks a construct signature, implicitly has an 'any' type.

16     container = new (getClass(classReference.prototype))()
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

My Typescript version is 3.8.3

Does anyone know what that could be?

**Comment by steelbrain** [comment]
My bad, I'll push a patch release

**Comment by steelbrain** [comment]
Should be fixed in v4.0.1
